packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -712,6 +729,44 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
         }
       : undefined;
 
+    /* @conditional-compile-remove(media-access) */
+    const onForbidParticipantAudio = async (userIds: string[]): Promise<void> => {
+      const participants = userIds?.map((userId) => _toCommunicationIdentifier(userId));
+      await call?.feature(Features.MediaAccess).forbidAudio(participants);
+    };
+    /* @conditional-compile-remove(media-access) */
+    const onPermitParticipantAudio = async (userIds: string[]): Promise<void> => {
+      const participants = userIds?.map((userId) => _toCommunicationIdentifier(userId));
+      await call?.feature(Features.MediaAccess).permitAudio(participants);
+    };
+    /* @conditional-compile-remove(media-access) */
+    const onForbidAllAttendeesAudio = async (): Promise<void> => {
+      await call?.feature(Features.MediaAccess).forbidRemoteParticipantsAudio();
+    };
+    /* @conditional-compile-remove(media-access) */
+    const onPermitAllAttendeesAudio = async (): Promise<void> => {
+      await call?.feature(Features.MediaAccess).permitRemoteParticipantsAudio();
+    };
+
+    /* @conditional-compile-remove(media-access) */
+    const onForbidParticipantVideo = async (userIds: string[]): Promise<void> => {
+      const participants = userIds?.map((userId) => _toCommunicationIdentifier(userId));
+      await call?.feature(Features.MediaAccess).forbidVideo(participants);
+    };
+    /* @conditional-compile-remove(media-access) */
+    const onPermitParticipantVideo = async (userIds: string[]): Promise<void> => {
+      const participants = userIds?.map((userId) => _toCommunicationIdentifier(userId));
+      await call?.feature(Features.MediaAccess).permitVideo(participants);
+    };
+    /* @conditional-compile-remove(media-access) */
+    const onForbidAllAttendeesVideo = async (): Promise<void> => {
+      await call?.feature(Features.MediaAccess).forbidRemoteParticipantsVideo();
+    };
+    /* @conditional-compile-remove(media-access) */
+    const onPermitAllAttendeesVideo = async (): Promise<void> => {
```

Comment:
I think we should rename 'onPermitAllAttendeesVideo ' to 'onPermitAllRemoteParticipantsVideo' and likewise for the other handlers like `onMuteAllRemoteParticipants ` to keep it consistent with the other handlers and the Calling SDK.

Reply:
It is not All remote participants, attendees only. 
Agree to keep consistent, will change to onPermitRemoteParticipantsVideo

Reply:
Changed it.

Reply:
I see. Only participants with attendee role will be affected. I still say to keep it consistent but we just have to document that only attendees will be affected.